src/python/pants/backend/jvm/tasks/jvm_compile/rsc/rsc_compile.py

```diff
@@ -454,10 +454,19 @@ def make_zinc_job(target, input_product_key, output_products, dep_keys):
         # Otherwise, fail it.
         on_success=ivts.update,
         on_failure=ivts.force_invalidate)
+        
+    workflow = rsc_compile_context.workflow
+
+    # Replica of JvmCompile's _record_target_stats logic
+    self.context.run_tracker.report_target_info(
+      self.options_scope,
+      compile_target,
+      ['compile', 'workflow'],
```

Comment:
could you also add `execution_strategy` to track in this change?

Reply:
(`self.execution_strategy_enum` would also work! if you check `NailgunTask` you'll see `self.execution_strategy` just accesses the `.value` (there's a TODO to fix it). i would probably suggest using `self.execution_strategy_enum` here)

Reply:
Added